purchase_order_maintenance_equipment_link/views/purchase_order_views.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="purchase_order_form" model="ir.ui.view">
+        <field name="name">purchase.order.form (in purchase_order_maintenance_equipment_link)</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='partner_ref']" position="before">
+                <field 
+                    name="maintenance_equipment_default_id"
+                    attrs="{'readonly': [('state', 'in', ('purchase', 'to approve','done', 'cancel'))]}" />
+            </xpath>
+            <xpath expr="//field[@name='product_id']" position="before">
+                <field 
+                    name="maintenance_equipment_ids"
+                    widget="many2many_tags"/>
```

Comment:
This field is shown, at least in `es` language, in view in singular, not plural, could you check?